osmaxx/conversion/converters/converter_gis/static/METADATA.txt

```diff
@@ -1,4 +1,4 @@
-titel               = "OpenStreetMap-Daten im Datenmodell Osmaxx v.2"
+titel               = "OpenStreetMap-Daten im Datenmodell OSMaxx v.2"
```

Comment:
Other stuff in here (e.g., whether "v.2" is still appropriate) will be handled in #636.